src/Net/HttpListenerResponseWrapper.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace Rothko
+{
+    public class HttpListenerResponseWrapper : IHttpListenerResponse
+    {
+        private HttpListenerResponse inner;
+
+        public HttpListenerResponseWrapper(HttpListenerResponse inner)
+        {
+            this.inner = inner;
+        }
+
+        ~HttpListenerResponseWrapper()
+        {
+            Dispose(false);
+        }
+
+        void IDisposable.Dispose()
```

Comment:
Is there a reason you're explicitly implementing this method? This should probably simply be `public void Dispose` and then you can remove the suppression.

Reply:
Just because in the original `HttpListener` it's (bizarrely) implemented like that. Now I know we don't have to follow the original API exactly, I'll make this look more normal.